.github/workflows/ci-test.yaml

```diff
@@ -86,3 +86,45 @@ jobs:
 
       - name: Test JS client
         run: pnpm test:unit
+
+  test-client-py:
+    name: test-client-py
+    runs-on: ubuntu-latest
+    services:
+      sapphire-localnet-ci:
+        image: ghcr.io/oasisprotocol/sapphire-localnet:latest
+        ports:
+          - 8545:8545
+          - 8546:8546
+        env:
+          OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
```

Comment:
I think sapphire-localnet behavior has been changed, so it's no longer necessary to override the `OASIS_DEPOSIT_BINARY` envvar, as it will generate the test mnemonics by default now.